fv3core/stencils/saturation_adjustment.py

```diff
@@ -527,7 +453,7 @@ def wqsat_correct(src, pt1, lhl, qv, ql, q_liq, q_sol, mc_air, c_vap):
 
 
 @gtstencil()
-def ap1_stencil(ta: sd, ap1: sd):
+def ap1_stencil(ta: FloatField, ap1: FloatField):
     with computation(PARALLEL), interval(...):
         ap1 = ap1_for_wqs2(ta)
```

Comment:
I think we can delete this? Is there anything left that is using it?

Reply:
Nothing's using it so I'll delete this as well.